syntax = "proto3";

package journey11;

message Task {
    string _task_name = 1;
    int64 _task_id = 2;
}